libs/libapi/src/libapi/response.py

```diff
@@ -58,3 +62,52 @@ def create_response(
         "num_rows_total": num_rows_total,
         "num_rows_per_page": MAX_NUM_ROWS_PER_PAGE,
     }
+
+
+def create_maybe_partial_response(
```

Comment:
wouldn't it make sense to add the partial field to PaginatedResponse?

Reply:
No because /rows always returns complete data (the rows from 0 to 100 are complete, whereas a search may ignore some rows because it only scans the first 5GB)

Reply:
We can set `partial: true` for `/rows`. Ideally, all the endpoints that return "rows" would use the same format.

Reply:
So `/rows` would return `partial: false` all the time.

Ah actually there is one case where it can return `partial: true`: if we request the last page and if we can't show the rest of the dataset.

Let me add `partial` to `/rows` too then

Reply:
I think I'll just add `partial: true` for all pages of `/rows` if the dataset is partially exported